server/src/main.rs

```diff
@@ -40,7 +40,7 @@ async fn main() -> io::Result<()> {
 
     HttpServer::new(|| {
         App::new()
-            .data(model::init(205693129))
+            .data(model::init(205_693_129))
```

Comment:
Why `205_693_129`? is that a valid numeric type in rust? 

I think i can continue with this moving forward as  our Model -> {games, random_seed}

Ill work on creating  a game object from the `post_game` function

Reply:
It is a valid format for numbers in Rust. Its the same value as if there were no underscores.

I made this change because it was a recommendation from `cargo clippy`.